### ISSUE TITLE
bin/plugin output for empty plugins isn't machine-friendly

### ISSUE THREAD

**Issue by drewr** [elastic/elasticsearch#8563]
The default output when you don't have any plugins:

```
% bin/plugin --list
Installed plugins:
    - No plugin detected in /usr/local/elasticsearch/elasticsearch-2.0.0-SNAPSHOT/plugins
```

looks too much like when you _do_ have plugins:

```
% bin/plugin --list
Installed plugins:
    - kibana
```

Can we just have a `\n`-separated list with nothing else?  And just no output if there aren't any installed?  Like so:

```
% bin/plugin --list
%
% bin/plugin -i elasticsearch/kibana/latest
% bin/plugin --list
kibana
%
```

At the very least can we change `- No plugin ...` to not have the same syntax as a plugin that _is_ installed?

**Comment by nellicus** [comment]
@tlrx 

can we have this to work in conjunction with --verbose?
today there is no diff

```
abonuccelli@w530 /opt/elk/PRODSEC/node1/elasticsearch-1.6.0/bin $ ./plugin -l
Installed plugins:
    - watcher
    - marvel
    - license
    - shield


abonuccelli@w530 /opt/elk/PRODSEC/node1/elasticsearch-1.6.0/bin $ ./plugin -l -v
Installed plugins:
    - watcher
    - marvel
    - license
    - shield
```

Can we have -v to print version and location of each plugin 

@drewr I disagree on "no output if there aren't any installed? " this is very user unfriendly.

**Comment by drewr** [comment]
If you type `ls` in an empty directory, you get no output.  If you list plugins and there aren't any installed why is no output not a valid response?  It's what makes things like `for i in $(bin/plugin -l); do ... done`  work.

**Comment by clintongormley** [comment]
Fixed in 5.0.0
